### ISSUE TITLE
Relevance for v2

### ISSUE THREAD

**Issue by Yomguithereal** [dekujs/magic-virtual-element#5]
Is this package meant to still be relevant to deku v2? Should we update it?

**Comment by anthonyshort** [comment]
Yeah it needs an update to return data in the same structure as the `element` function. Then you could use it as a drop-in replacement.

**Comment by askmatey** [comment]
See #6

**Comment by askmatey** [comment]
Issue resolved with `2.0.0-rc1`.
